i18n/nl_NL.csv

```diff
@@ -117,6 +119,8 @@ If both have been selected this price will be added to the regular shipping pric
 "Besides assigning weights to all products, you can also specify a volume of each product.","Om deze functie optimaal te gebruiken, stel je eerst het gewicht of volume in van jouw producten."
 This will be added to the regular shipping price,Dit zal bij de normale verzendkosten worden opgeteld
 Pickup at Bpost location,Ophalen bij een Bpost locatie
+List,Lijstweergave
+Map,Kaartweergave
```

Comment:
lijst/kaart is beter, het is de tekst in de knopjes en dit wordt te lang

Reply:
Done